attachment_synchronize/models/attachment_queue.py

```diff
@@ -21,6 +21,11 @@ class AttachmentQueue(models.Model):
         selection_add=[("export", "Export File (External location)")]
     )
 
+    def _write_file_to_remote(self, fs, full_path, data):
+        self.ensure_one()
```

Comment:
`self` is not used in this method, so I don't understand why the constraint `self.ensure_one()`.
I'd suggest to either remove it and use `api.model`
```suggestion
    @api.model
    def _write_file_to_remote(self, fs, full_path, data):
```
or actually use it to retrieve `data`
```suggestion
    def _write_file_to_remote(self, fs, full_path):
        self.ensure_one()
        data = base64.b64decode(self.datas)
```
what do you think?

Reply:
I think the `self.ensure_one` is important, because the aims of this hook is to be able to override it so do custom stuff that would depend on the attachment queue object.

But, I agree that the data could be in this method, make sense IMO
I just done it